src/components/Alert/index.jsx

```diff
@@ -2,5 +2,5 @@
 Copyright (C) 2018 The Trustees of Indiana University
 SPDX-License-Identifier: BSD-3-Clause
 */
-export { default as Alert } from "./Alert";
-export { default as DismissibleAlert } from "./DismissibleAlert";
+export { default as Alert } from "./Alert.jsx";
+export { default as DismissibleAlert } from "./DismissibleAlert.jsx";
```

Comment:
Is this something prettier did? We don't have file extensions on other internal imports/exports.

Reply:
No, but for some reason styleguidist wouldn't run this without the suffix. I checked and it isn't happening now. Weird. Will remove it.